rclpy/src/rclpy/graph.cpp

```diff
@@ -249,4 +250,75 @@ graph_get_service_names_and_types(py::capsule pynode)
   return convert_to_py_names_and_types(&service_names_and_types);
 }
 
+typedef rcl_ret_t (* rcl_get_info_by_topic_func_t)(
+  const rcl_node_t * node,
+  rcutils_allocator_t * allocator,
+  const char * topic_name,
+  bool no_mangle,
+  rcl_topic_endpoint_info_array_t * info_array);
+
+py::list
+_get_info_by_topic(
+  py::capsule pynode,
+  const char * topic_name,
+  bool no_mangle,
+  const char * type,
+  rcl_get_info_by_topic_func_t rcl_get_info_by_topic)
+{
+  (void) type;
```

Comment:
@hidmic I had to void this as it's no longer used in the new error messages. Please let me know if I should remove it or how to re-write the errors to incorporate it.

Reply:
Hmm, `std::string` can be concatenated. There's also `std::sstringstream`.